### DISCUSSION TITLE
Votekick Commanders

### DISCUSSION THREAD

**Discussion by Cornholio777** [data-bomb/Silica#24] [Ideas]
I have an idea for a new mod that would improve the experience for players on the servers.
!votekickcmdr playername

This command would only boot the commander from the Commander seat, not the match.
A big issues are new commanders on busy servers, a lot of players feel their experience is ruined by a new commander who have not learned the basics yet of commanding.
A majority vote to boot the commander from the commander seat, maybe just for a little while so another can join:
!demote exists but it's a perma ban from commanding and I have to be around to do it. I could always return but an votekick system would players be able to choose their own commander.
Ofc I think anyone who is new to commanding can try, but they should get experience first in singleplayer and then join the multiplayers as commander.

**Comment by data-bomb** [comment]
!demote is a one-time temporary admin action and can be used without worrying about preventing the player from becoming commander in the future. A mutiny system is planned but there is not a feasible way to implement that right now because there is no votes or vote UI in the base game.

**Comment by Cornholio777** [comment]
I see, in that case I guess it has to wait until the developer adds in a vote UI in the base game.
